tendrl/gluster_integration/sds_sync/utilization.py

```diff
@@ -33,6 +33,9 @@ def sync_utilization_details(volumes):
             volume.usable_capacity = int(util_det['total'])
             volume.used_capacity = int(util_det['used'])
             volume.pcnt_used = str(util_det['pcnt_used'])
+            volume.usable_inode_capacity = int(util_det['total_inode'])
+            volume.used_inode_capacity = int(util_det['used_inode'])
+            volume.pcnt_inode_used = str(util_det['pcnt_inode_used'])
```

Comment:
why using str for percentage?

Reply:
As of now, obj attrs cannot be of float type. So to preserve the precision str is being used